tests/unit/preprocessor/_time/test_time.py

```diff
@@ -1855,6 +1855,31 @@ def make_map_data(number_years=2):
         )
 
 
+@pytest.mark.parametrize("period", ["full"])
+def test_relative_anomalies(period):
+    """Test  relative ``anomalies``."""
+    cube = make_map_data(number_years=2)
+    result = anomalies(cube, period, relative=True)
+    if period == "full":
+        expected_anomalies = cube.data - np.mean(
+            cube.data,
+            axis=0,
+            keepdims=True,
+        )
+        expected_meananomalies = (
+            expected_anomalies
+            / np.mean(
+                cube.data,
+                axis=0,
+                keepdims=True,
+            )
+            * 100.0
+        )
+        expected = np.ma.masked_invalid(expected_meananomalies)
+        assert_array_equal(result.data, expected)
+        assert result.units == "%"
```

Comment:
@axel-lauer have a look at the codecov's [coverage output](https://app.codecov.io/gh/ESMValGroup/ESMValCore/pull/2871?dropdown=coverage&src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=checks&utm_campaign=pr+comments&utm_term=ESMValGroup) - there are a few lines that are not covered by tests - it'd be great if you could add a unit test for the new private `_apply_scaling()` func; let me know if I can help with test writing 🍺